docker_rsync.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+from os.path import abspath, join
+import docker
+
+from logger import log_from_docker
+
+client = docker.from_env()
+
+
+def run_rsync(volumes, from_path, to_path, relative):
+    # rsync options:
+    # -v = verbose - give info about what files are being transferred and a brief summary at the end
+    # -r = copy directories recursively
+    # -e = specify remote shell program explicitly (i.e. ssh as opposed to the default rsh)
+    # --delete = delete destination files not in source
+    cmd = ["rsync", "-rv", "-e", "ssh", from_path, to_path, "--delete"]
+
+    if relative:
+        cmd = ["rsync", "-rv", "-e", "ssh", "--relative", from_path, to_path, "--delete"]
+
+    container = client.containers.run("instrumentisto/rsync-ssh", command=cmd, volumes=volumes,
+                                      detach=True)
+
+    log_from_docker(container)
+
+
+def create_volumes(settings, local_volume, volume_mode):
+    mounted_volume = join("/", local_volume)
+    return {abspath(settings.known_hosts_path): {"bind": "/root/.ssh/known_hosts", "mode": "rw"},
```

Comment:
I'm finding this a bit hard to read. Could we maybe:

1. Space it out, so the outermost dictionary gets a line for each of its braces?
1. Move the `abspath` call into settings? Do we ever need them as relative paths?
1. Replace `"{}".format(mounted_volume)` with just `mounted_volume`